### PR TITLE
Improve lodestar-config exports

### PR DESCRIPTION
**Pull request by dapplion**
Prevent having to import from /lib and deliver what the README promises.

Adds two files in the root of lodestar-config that re-export the preset files

### CODE REVIEW

File: packages/lodestar-config/README.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ Configuration variables for an Eth2 network -- consists of params and ssz types
 
 ```typescript
 // mainet and minimal presets are available under non-default exports
-import {config as mainnetConfig} from "@chainsafe/lodestar-config/lib/presets/mainnet";
-import {config as minimalConfig} from "@chainsafe/lodestar-config/lib/presets/mainnet";
+import {mainnetConfig} from "@chainsafe/lodestar-config/mainnet";
+import {minimalConfig} from "@chainsafe/lodestar-config/mainnet";
```
Comment (twoeths) [inline packages/lodestar-config/README.md]: `import {config} from "@chainsafe/lodestar-config/mainnet";`?

File: packages/lodestar-config/mainnet.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+export * from "./lib/presets/mainnet";
```
Comment (twoeths) [inline packages/lodestar-config/mainnet.d.ts:1]: can we have `lodestar-config/mainnet.ts` instead?

File: packages/lodestar-config/README.md (hunk 1 above)
Comment (dapplion) [reply packages/lodestar-config/README.md]: Do you suggest to only use a single import statement in the README example?

File: packages/lodestar-config/mainnet.d.ts (hunk 2 above)
Comment (dapplion) [reply packages/lodestar-config/mainnet.d.ts:1]: That would require compiling code outside of src, I haven't found a clean way to do it

File: packages/lodestar-config/README.md (hunk 1 above)
Comment (twoeths) [reply packages/lodestar-config/README.md]: ah I just mean the change you did everywhere is different from this README :D

**Review by wemeetagain — DISMISSED** [review]

**Comment by wemeetagain** [comment]
file conflict

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**